core/sockets.py

```diff
@@ -264,6 +265,15 @@ def draw_menu_items(self, context, layout):
         if self.can_wrap():
             layout.prop(self, 'use_wrap')
 
+def format_property(prop):
+    if isinstance(prop, (str, int, float)):
+        return [[prop]]
+    if hasattr(prop, '__len__'):
+        # it looks like as some BLender property array - convert to tuple
+        return [[prop[:]]]
+
+    return [prop]
```

Comment:
```Python
from sverchok.utils.handle_blender_data import BPYProperty
prop_value = PBYProperty(socket, prop_name).value
```
I don't think this is the place for the function anyway, probably move into some utils module.

Reply:
I dont get what are you proposing

Reply:
I think it's better to move the function into some another (utility) module for keeping sockets module clean. It's actually already too large now.